docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Centers.md

```diff
@@ -0,0 +1,45 @@
+# Centers
+
+## Overview Study
+
+**Sites** or **Centers** are terms used interchangeably in LORIS.
+**Centers** are defined in the `psc` table of the database 
+(_psc_ stands for _Project Study Center_).
+
+LORIS' default schema defines the first `psc` (`CenterID=1`) as the
+Data Coordinating Center (DCC).  This center is generally used for
+dummy data and it is assumed by the codebase to store exclusively 
+non-study data. Therefore, it is not recommended to modify or use 
+this center for registering real participants or study data.
+
+>  Note: There cannot be more than one center with the same name.
+
+## Adding Site Options
+
+### Front End
+
+_not yet available_
+
+### SQL
+
+The `psc` table in the database contains all the information stored
+for centers. Populate additional centers using the following MySQL
+command:
+
+```sql 
+INSERT INTO psc (Name, Alias, MRI_alias, Study_site) VALUES ('Montreal','MTL','MTL','Y');
+```
+
+> Please refer to the [SQL Dictionary]() for additional information about
+> this table and its fields.
+
+### API
+
+ _not yet available. See [API documentation](../../../API/) for latest
```

Comment:
Maybe mention that you can get a candidate's centers, but not the project's centers?

Reply:
this is for set up
you cant currently add projects from the API no ?